RELEASE/scripts/autoscend/paths/actually_ed_the_undying.ash

```diff
@@ -1034,6 +1034,9 @@ void ed_handleAdventureServant(location loc)
 
 boolean L1_ed_island()
 {
+	//reset tracking of Ka farming
+	set_property("_auto_farmingKaAsEd", false);
```

Comment:
```suggestion
	remove_property("_auto_farmingKaAsEd");
```
I think it would make for a cleaner log to use remove property instead of changing it from true to false